Externals.cfg

```diff
@@ -3,7 +3,7 @@ protocol = git
 repo_url = https://github.com/ufs-community/regional_workflow
 # Specify either a branch name or a hash but not both.
 #branch = develop
-hash = 405553b
+hash = 3f83cde
```

Comment:
Are you updating all the hashes to the latest?  If so, I strongly suggest doing a full set of tests.  I think @mkavulich was doing similar and found a problem with the latest weather model and SPP.

Reply:
Yeah, it'd be good to run a full set of tests with all the hashes being updated.  I think the problem was specifically related to the build on Cheyenne with ESMF, so SPP should be fine.

Reply:
Oh, I had not exactly intended to jump ahead. I pulled the hashes from the current head of develop. Where should they be pulled from?

Reply:
The alternative would be just to keep the hashes that were in Externals.cfg.

Reply:
The hashes that were in Externals.cfg (e.g. 405553b)  were from when I first started this PR, so they were out of date. The hashes that are in there now are the same as those in the head of develop. I am not exactly sure why they are showing up as changed.